README.md

```diff
@@ -4,30 +4,30 @@
 
 **Specifications:**
 
-- [ ] The user should be able to register
-  - [ ] Username
-  - [ ] Password (Min length of 6, should contain numbers and letters)
-  - [ ] First and Last Name
-  - [ ] Email (in email format)
-- [ ] The user should be able to login after successful registration via email and password
-- [ ] Upon login, the user should be able to see a dashboard on the first page
-  - [ ] Current balance of the wallet
-  - [ ] Option to add amount to the balance
-  - [ ] Option to pay
-  - [ ] Transaction history (latest 5)
-  - [ ] Menu Options
-    * [ ] Transaction History (Full list)
-    * [ ] Saved contact List
-    * [ ] Logout
-- [ ] Option to add amount to the balance
-  - [ ] Amount
-  - [ ] Description
-  - [ ] Date of Transaction (read-only)
-- [ ] Option to pay
-  - [ ] Through QR code (or search inside contact list)
-  - [ ] Amount
-  - [ ] Description
-  - [ ] Date of Transaction (read-only)
-- [ ] Saved Contact List
-  - [ ] The user should have an option to save users to the contact list after the payment has been made for easy future transactions
-  - [ ] The list should contain option to pay again to that person
+- [X] The user should be able to register
```

Comment:
It's good you have readme!

Reply:
Thanks. I actually forgot to add here how to run the program. It's actually useful when someone who doesn't use flutter tries to run the program.